src/db_impl.cc

```diff
@@ -1015,6 +1045,11 @@ bool TitanDBImpl::GetIntProperty(ColumnFamilyHandle* column_family,
 }
 
 void TitanDBImpl::OnFlushCompleted(const FlushJobInfo& flush_job_info) {
+  TEST_SYNC_POINT("TitanDBImpl::OnFlushCompleted:Begin");
+  if (!initialized()) {
```

Comment:
why check twice in basedb listener and here?

Reply:
just to be safe.